lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -129,7 +127,7 @@ def local_output_dir(windows_dir)
         # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
         # Software and version used to do OCR processing, found in ALTO output
         def processing_metadata
-          return {} unless File.exist? alto_doc # PDFs don't have ALTO, so just bail out
+          return {} unless alto_doc && File.exist?(alto_doc) # PDFs don't have ALTO, so just bail out
```

Comment:
this change will stop this HB alert from being triggered needlessly: https://app.honeybadger.io/projects/52894/faults/111166939